core/templates/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -29,18 +29,23 @@ import { LoaderService } from 'services/loader.service';
 import { LoggerService } from 'services/contextual/logger.service';
 import { ExplorationChange } from 'domain/exploration/exploration-draft.model';
 import { WindowRef } from 'services/contextual/window-ref.service';
+import { ConnectionService } from 'services/connection-service.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ChangeListService implements OnInit {
+  hasNetworkConnection: boolean;
+  hasInternetAccess: boolean;
+  status: string;
   // Temporary buffer for changes made to the exploration.
   explorationChangeList: ExplorationChange[] = [];
   undoneChangeStack: ExplorationChange[] = [];
   // Stack for storing undone changes. The last element is the most recently
   // undone change.
   ndoneChangeStack = [];
   loadingMessage: string = '';
+  tempChanges: ExplorationChange[] = [];
```

Comment:
What are `tempChanges`? Also, we can use full English words when possible.

Reply:
changed the variable to ```temporaryListOfChanges``` and also added a comment to explain it.